### PR TITLE
chore: add checklist to PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by YoshinoriN**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,20 @@
-- [ ] Read the [theme publishing doc](https://hexo.io/docs/themes#Publishing) or [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
+## Check List
+
+**Please read and check followings before submitting a PR.**
+
+- [ ] I want to publish my theme on Hexo official website.
+    - [ ] I have already read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
```
Comment (curbengh) [inline .github/PULL_REQUEST_TEMPLATE.md]: "I have read...", applies to the rest.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+    - [ ] `link` URL is correct.
+    - [ ] `preview` URL is correct.
+    - [ ] `preview` URL web site is render correctly.
```
Comment (curbengh) [inline .github/PULL_REQUEST_TEMPLATE.md]: "...is rendered..."

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,4 +1,20 @@
...
+    - [ ] Add a screenshot to `source/themes/screenshots`.
+    - [ ] Screenshot file name with the same as `name`.
+    - [ ] Screenshot size is `800 * 500`.
+    - [ ] Screenshot file format is `png`.
+- [ ] I want to publish my plugin on Hexo official website.
+    - [ ] I have already read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
+    - [ ] `link` URL is correct.
+- [ ] Others (Update, fix, transrate, etc...)
```
Comment (curbengh) [inline .github/PULL_REQUEST_TEMPLATE.md]: "translation"

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (YoshinoriN) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thanks :)
I will fix later. Also others.

**Comment by YoshinoriN** [comment]
Updated

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,4 +1,20 @@
...
+
+**Please read and check followings before submitting a PR.**
+
+- [ ] I want to publish my theme on Hexo official website.
+    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
+    - [ ] `link` URL is correct.
+    - [ ] `preview` URL is correct.
+    - [ ] `preview` URL web site is rendered correctly.
+    - [ ] Add a screenshot to `source/themes/screenshots`.
+    - [ ] Screenshot file name with the same as `name`.
```
Comment (curbengh) [inline .github/PULL_REQUEST_TEMPLATE.md]: "...filename is same as value of `name`..."

Comment (YoshinoriN) [reply .github/PULL_REQUEST_TEMPLATE.md]: done :)

**Comment by YoshinoriN** [comment]
@curbengh 
Many thanks. I need to improve my English... 😓

**Pull request merged**